styles/global.scss

```diff
@@ -257,6 +257,14 @@ hr {
   }
 }
 
+a.has-text-grey {
```

Comment:
hmmm looks weird but ok

Reply:
Imo this can have side effects, maybe writte it within `.connect-wallet-modal` class (scoped component class)

Reply:
I found out that this applies everywhere. I put it here just in case we also have that class in a different place. wdyt?

for example, you can reproduce that by adding `has-text-gray` class to our footer link

Reply:
I though you changed line 254, alright let it roll like this